packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -86,93 +87,137 @@ export default function ScheduleInputs(props: Props) {
-                  </span>
-                  <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
-                    <SelectField
-                      name="date-operator"
-                      value={
-                        timestamp !== null
-                          ? "at a specific date and time"
-                          : enableFeature
-                          ? "immediately"
-                          : "manually"
-                      }
-                      options={(i + 1) % 2 === 1 ? launchOptions : endOptions}
-                      onChange={(value) => {
-                        if (value === "at a specific date and time") {
-                          onChange(new Date().toISOString(), "timestamp", i);
-                        } else {
-                          onChange(null, "timestamp", i);
-                        }
+          <li className={styles.listitem}>
+            <div className="row align-items-center">
+              <span className="ml-2 mb-2">Launch rule</span>
+              <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
+                <SelectField
+                  name="date-operator"
+                  value={
+                    rules[0].timestamp === null
+                      ? "immediately"
+                      : "at a specific date and time"
+                  }
+                  options={launchOptions}
+                  onChange={(value) => {
+                    if (value === "at a specific date and time") {
+                      onChange(
+                        format(new Date(), "yyyy-MM-dd'T'HH:mm"),
+                        "timestamp",
+                        0
+                      );
+                    } else {
+                      onChange(null, "timestamp", 0);
+                    }
+                  }}
+                />
+              </div>
+              {rules[0].timestamp !== null && (
+                <>
+                  <div className="w-auto mb-2 p-2">
+                    <span className="mb-2">ON</span>
+                  </div>
+                  <div className="col-sm-12 col-md mb-2 d-flex align-items-center">
+                    <Field
+                      type="datetime-local"
+                      className={clsx(dateErrors[0] && styles.error)}
+                      value={format(
+                        new Date(rules[0].timestamp),
+                        "yyyy-MM-dd'T'HH:mm"
+                      )}
+                      onChange={(e) => {
+                        onChange(e.target.value, "timestamp", 0);
                       }}
+                      name="timestamp"
                     />
+                    <span className="pl-2">
+                      (
+                      {new Date()
+                        .toLocaleDateString(undefined, {
+                          day: "2-digit",
+                          timeZoneName: "short",
+                        })
+                        .substring(4)}
```

Comment:
This doesn't work for all locales.  For example, in South Korea (`ko-KR`), a "2-digit" day is actually 3 characters long so your substring will be incorrect.  Timezones and locales have so many edge cases I don't recommend touching them at all without a proper date library.

Reply:
I'm doing something similar within the `RuleStatusPill` component - I'll update that there, as well.

Reply:
Updated to use the `date-fns-tz` library.